test/battery/accessibility/odrv_test.toml

```diff
@@ -0,0 +1,24 @@
+# [[source]]
+#     path = "src.cpp"
+#     object_file_name = "public"
+#     flags = [
+#         "-D'ORC_TEST_ACCESS=public'"
+#     ]
+
+[[source]]
+    path = "src.cpp"
+    object_file_name = "private"
+    flags = [
+        "-D'ORC_TEST_ACCESS=private'"
+    ]
+
+[[source]]
+    path = "src.cpp"
+    object_file_name = "protected"
+    flags = [
+        "-D'ORC_TEST_ACCESS=protected'"
+    ]
+
+[[odrv]]
```

Comment:
I love this new structure, and binding the tests & toml together.
Question: should the expected ODRV be in the [[source]] block? Could different variants create different ODRVs? Or would we expect that to almost always mean you need separate tests?

Reply:
I'd want to see separate tests for separate variants. My vision right now is one ODRV per unit test, if possible. I'm open to flex on that, if it makes sense to do so. But keeping them singular makes it straightforward to enforce/debug.

Reply:
also makes sense!